src/internal/operators/merge.ts

```diff
@@ -23,7 +22,6 @@ export function merge<T, A extends readonly unknown[]>(
 export function merge<T>(...args: unknown[]): OperatorFunction<T, unknown> {
   const scheduler = popScheduler(args);
   const concurrent = popNumber(args, Infinity);
-  args = argsOrArgArray(args);
```

Comment:
This was clearly a mistake, because these `args` are being spread inside the from afterwords, so we always want the array, even when it only has one value.

Reply:
The typings on line 9 show that we were trying to handle the case of `mergeWith([a$, b$, c$])`...

I'm on the fence here. There are two issues at play:

1. `a$.pipe(mergeWith(b$, c$))` is more ergonomic
2. The `concurrency` limit should be provided via a configuration object at the end at some point, which means that `mergeWith([...], { concurrency: 2 })` is a LOT cleaner to implement, but `mergeWith(..., { concurrency: 2 })` is fine as well just obnoxious to implement.

Also, it's technically a breaking change from intent, but there's clearly been a bug that didn't always allow the intent.... and no one has complained. 🤔

Reply:
You're right... I was just getting back to this, and even dating back to RxJS 5, we didn't support `[a$, b$, c$]` as an argument. I'm probably just thinking of `combineLatest` or something else.